### PR TITLE
Update position.json

### PR DESCRIPTION
**Pull request by Keyacom**
#### Summary

Added info in `.../position_sticky_table_elements/__compat/support/firefox/notes` about a bug.
#### Test results and supporting details

[Bug 1727594](https://bugzilla.mozilla.org/show_bug.cgi?id=1727594).

### CODE REVIEW

File: css/properties/position.json [hunk 1]
```diff
@@ -171,6 +171,7 @@
               },
               "firefox": {
                 "version_added": "59"
```
Comment (queengooborg) [inline css/properties/position.json:173]:
```suggestion
                "version_added": "59",
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Hey there, thank you for your PR, and welcome to BCD!  Before I properly review your PR, I noticed that there's a syntax error in the JSON.

File: css/properties/position.json (hunk 1 above)
Comment (Keyacom) [reply css/properties/position.json:173]: Sorry, I just didn't notice it, but thank you anyways.

**Review by queengooborg — APPROVED** [review]
Thanks for the fixes!

I think we should also add the note to Firefox Android -- I'm seeing no indicating that the issue isn't also on Android.

**Pull request merged**